controllers/imagepolicy_controller.go

```diff
@@ -180,6 +180,14 @@ func (r *ImagePolicyReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		var tags []string
 		tags, err = r.Database.Tags(repo.Status.CanonicalImageName)
 		if err == nil {
+			if len(tags) == 0 {
+				msg := fmt.Sprintf("no tags found in '%s', retrying in %s", repo.Name, repo.Spec.Interval.Duration)
```

Comment:
Maybe we should mention where we didn’t found them?
```suggestion
				msg := fmt.Sprintf("no tags found in local storage for '%s', retrying in %s", repo.Name, repo.Spec.Interval.Duration)
```